Makefile

```diff
@@ -16,7 +16,7 @@ export GO111MODULE := on
 GOOS := $(if $(GOOS),$(GOOS),linux)
 GOARCH := $(if $(GOARCH),$(GOARCH),amd64)
 GOENV  := GO15VENDOREXPERIMENT="1" CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH)
-GO     := $(GOENV) go build
+GO     := $(GOENV) go build -trimpath
```

Comment:
Please change this line too: https://github.com/pingcap/tidb-operator/pull/1129/files#diff-b67911656ef5d18c4ae36cb6741b7965L11